mysql-test/suite/versioning/t/alter.test

```diff
@@ -282,6 +282,21 @@ select * from t for system_time all;
 alter table t add column b int;
 select * from t for system_time all;
 
+create or replace table non_empty (
+  a int,
+  sys_trx_start bigint(20) unsigned,
+  sys_trx_end bigint(20) unsigned
+) engine innodb;
+insert into non_empty values (1, 100, 200);
```

Comment:
Why 3 values? Use as low data as possible.

Reply:
It's one tuple with 3 values. Note parentheses.